src/app/components/gallery/submit/submit-ncj-template.component.spec.ts

```diff
@@ -282,4 +286,19 @@ describe("SubmitNcjTemplateComponent", () => {
             expect(component.modeState).toBe(NcjTemplateMode.NewPoolAndJob);
         });
     });
+
+    describe("Submit", () => {
+        beforeEach(() => {
+            fixture = TestBed.createComponent(TestComponent);
+            de = fixture.debugElement.query(By.css("bl-submit-ncj-template"));
+            component = de.componentInstance;
+            fixture.detectChanges();
+        });
+
+        it("Should run telemetry", () => {
+            component.modeState = NcjTemplateMode.NewPool;
+            component.submit();
+            expect(telemetryServiceSpy.trackEvent).toHaveBeenCalledOnce();
```

Comment:
Can we add an assert on the telemetry contents here? (ie: check the type, templateMode)